src/main/java/org/folio/list/services/export/CsvCreator.java

```diff
@@ -80,7 +82,15 @@ public ExportLocalStorage createAndUploadCSV(ExportDetails exportDetails, String
       ContentsRequest contentsRequest = new ContentsRequest().entityTypeId(list.getEntityTypeId())
         .fields(exportDetails.getFields())
         .ids(ids);
-      var sortedContents = queryClient.getContents(contentsRequest);
+      var sortedContents = queryClient.getContents(contentsRequest)
+        .stream()
+        .filter(map -> !Boolean.TRUE.equals(map.get(IS_DELETED)))
+        .map(map -> {
+          // Remove _deleted key from the map if present
```

Comment:
I'm confused about this part. We shouldn't ever need to remove a _deleted key should we? We should just be removing maps that have _deleted = true, right?